prep/ARIA_prep.ipynb

```diff
@@ -9,7 +9,8 @@
     "\n",
```

Comment:
<p>Will this fail if we run for sites outside of North America where there is no HRRRStack?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
the latest changes shows whether the file exists or not. The `prep_aria.py` command will simply skip the non-existent files even though they may be set.

Reply:
<p>Great! thanks.</p>